src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/NotificationBannerController.kt

```diff
@@ -11,28 +10,22 @@ import uk.gov.justice.digital.hmpps.manageusersapi.service.NotificationBannerSer
 @RestController
 class NotificationBannerController(
   private val notificationBannerService: NotificationBannerService,
-  @Value("\${notification.banner.folder}") private val notificationBannerEnv: String,
 ) {
 
-  @GetMapping("/notification/banner/{type}", produces = [MediaType.APPLICATION_JSON_VALUE])
+  @GetMapping("/notification/banner/{page}", produces = [MediaType.APPLICATION_JSON_VALUE])
   fun getRoleBannerMessage(
```

Comment:
Could rename this `getBannerMessage` as it can return different types of banner messages not just for roles